src/test/fixtures/profiles/processed-profile.js

```diff
@@ -390,39 +400,6 @@ function _buildThreadFromTextOnlyStacks(
     funcTable.length++;
   });
 
-  // Go back through and create resources as needed.
-  funcNames.forEach(funcName => {
-    // See if this sample has a resource like "funcName:libraryName".
-    const [, libraryName] = funcName.match(/\w+:(\w+)/) || [];
-    let resourceIndex = resourceIndexCache[libraryName];
-    if (resourceIndex === undefined) {
-      const libIndex = libs.length;
-      if (libraryName) {
-        libs.push({
-          start: 0,
-          end: 0,
-          offset: 0,
-          arch: '',
-          name: libraryName,
-          path: '/path/to/' + libraryName,
-          debugName: libraryName,
-          debugPath: '/path/to/' + libraryName,
-          breakpadId: 'SOMETHING_FAKE',
-        });
-        resourceIndex = resourceTable.length++;
-        resourceTable.lib.push(libIndex);
-        resourceTable.name.push(stringTable.indexForString(libraryName));
-        resourceTable.type.push(resourceTypes.library);
-        resourceTable.host.push(undefined);
-      } else {
-        resourceIndex = -1;
-      }
-      resourceIndexCache[libraryName] = resourceIndex;
-    }
-
-    funcTable.resource.push(resourceIndex);
-  });
-
```

Comment:
I mostly just moved this in a different place. I very slightly changed some logic (like checking `libraryName` before, or getting rid of the `libIndex` intermediate variable).